packages/chart/src/components/SparkLine.js

```diff
@@ -90,10 +90,19 @@ export default function SparkLine({ width: parentWidth, height: parentHeight })
 
   const handleSparkLineTicks = [xScale.domain()[0], xScale.domain()[xScale.domain().length - 1]]
 
+  const isNumber = value => {
+    return value !== null && value !== '' && typeof value === 'number'
+  }
+
   useEffect(() => {
     ReactTooltip.rebuild()
   })
 
+  const getYscaleLog = value => {
```

Comment:
Great way of handling the debugging. However, we don't want to console log values every time the chart loads. Consider removing or adding a ternary on the log.

Reply:
Hmmm didnt realize I changed SparkLine.  I can remove.

Reply:
Removed this entirely.  Not needed for the repo.